### ISSUE TITLE
Sprint J Goals

### ISSUE THREAD

**Issue by dianagriffin** [department-of-veterans-affairs/abd-vro#2273]

**Comment by agile-josiah** [comment]
12/12
* Mason (5)
    * 2294
        * In Progress and stalled
        * Knows some of what we may need to cover, and wants to refine more
    * 2172
        * Merged
        * Wants to use a testing framework, but bugs is not following a good pattern to make it testable by ruby frameworks.
        * Not in scope of the ticket to refactor. Current solution is fine.
        * (Note from Teja: spike out what work is necessary to use a testing framework)
    * Working on RDS while waiting for 2294
* Diana (6-7)
    * Continuing to work with Paul and Megan on priorities 
    * Need Paul to get access to Slack
    * Sent an email to PIA to confirm we have what we need to store the contention field for the cc team
* Ponnia (4)
    * Spinning up services and trying to understand the app content.
    * Not successful at starting ee-merge app
    * Meeting with engineers to go into the services
* Josiah (9)
    * Worked on Secrel and had help from Mason to unblock deployments
* Megan (7)
    * Working with Diana and Paul on product stuff
    * Pinging for travel questions
    * Will go to Cor this afternoon
    * A lot of action on the vro channels regarding old PRs. Please pick up prs
    * Email regarding secrets, tagged Teja to coordinate a response. (16th miniute item)
* Teja (8)
    * Helping with BIP and db-init
    * Deployed some changes to Kafka
    * Will add Ponnia to the PR review team on GitHub
* Erik (6)
    * Working on Bip
        * Getting config changes to show up in dev
        * Pushed some changes this morning and will verify later this afternoon
        * Will reschedule bip meeting to finish working on deploying bip (potentially after the holidays)
* Bianca (5)
    * Filling out the survey form for partner teams.
        * By EOD today so Megan and Diana can review.
    * Shared 2023 narrative w/ Julie on ways ew can tell stories about the work we do.
* Cheng (8)
    * Creating a new ticket to address the CMNT field (#2346)
        * What is the name they want us to use?
    * Helped Erik debug the bip service in the kub pods
    * 1 PR for the automated script to generate secrets from BIP
* Paul (7)
    * Requesting a background investigation, has been rather challenging and time-consuming
    * Meeting to discuss prioritization w/ Diana and Megan
